### PR TITLE
Show accept request dialog when click in request on manage member page

### PR DESCRIPTION
**Pull request by Luiz-FS**
**Feature/Bug description:**  When you receive a request to be a member of an institution and enter the membership management page to accept, the form of acceptance is different from when the user clicks on the notification.

screenshot from 2018-04-17 10-56-36

**Solution:** I've changed how to accept requests to be a member of the member management page so that it complies with the notification.

screenshot from 2018-04-17 10-55-05

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/institution/managementMembersController.js [hunk 1]
```diff
@@ -67,6 +67,31 @@
             }, function error() {});
         };
 
+        manageMemberCtrl.openAcceptRequestDialog = function openAcceptRequestDialog(requestKey) {
```
Comment (raonismaneoto) [inline frontend/institution/managementMembersController.js]: you've forgotten to receive event as a param

File: frontend/institution/managementMembersController.js [hunk 2]
```diff
@@ -67,6 +67,31 @@
...
+                locals: {key: requestKey},
+                openFrom: '#fab-new-post',
+                closeTo: angular.element(document.querySelector('#fab-new-post'))
+            }).then(function success(processedRequest) {
+                const request = manageMemberCtrl.requests.reduce(
+                    (found, request) => (request.key === processedRequest.key) ? request : found, 
+                    {}
+                );
+                request.status = processedRequest.status;
+            }, function error() {
```
Comment (raonismaneoto) [inline frontend/institution/managementMembersController.js]: show some message to the user

File: frontend/institution/managementMembersController.js (hunk 1 above)
Comment (raonismaneoto) [inline frontend/institution/managementMembersController.js]: test this function and all others that hasn't been tested yet

File: frontend/institution/managementMembersController.js (hunk 2 above)
Comment (Luiz-FS) [reply frontend/institution/managementMembersController.js]: Is already being displayed

**Pull request merged**